### PR TITLE
fix: Q2 Dep Bumps 🌲

### PR DESCRIPTION
**Pull request by stdavis**
Including storybook major bump

### CODE REVIEW

File: .storybook/main.cjs [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
   stories: ['../packages/*/src/*.stories.@(js|jsx)'],
-  addons: [
-    '@storybook/addon-links',
-    '@storybook/addon-essentials',
-    '@storybook/addon-interactions',
-  ],
-  framework: '@storybook/react',
-  core: {
-    builder: '@storybook/builder-vite',
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-interactions', '@storybook/addon-mdx-gfm'],
```
Comment (steveoh) [inline .storybook/main.cjs:3]: i don't think we need the github flavored markdown package

**Review by steveoh — DISMISSED** [review]

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      - name: 🚀 Create Release
+        id: release-please
+        uses: agrc/release-composite-action@v1
+        with:
+          prerelease: ${{ github.ref_name == 'dev' }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          github-app-id: ${{ secrets.UGRC_RELEASE_BOT_APP_ID }}
+          github-app-key: ${{ secrets.UGRC_RELEASE_BOT_APP_KEY }}
+          github-app-name: ${{ secrets.UGRC_RELEASE_BOT_NAME }}
+          github-app-email: ${{ secrets.UGRC_RELEASE_BOT_EMAIL }}
```
Comment (steveoh) [inline .github/workflows/push.yml:35]: have we tested this on a mono repo?

Comment (stdavis) [reply .github/workflows/push.yml:35]: Oh, ya. This is probably a bad idea...

Comment (stdavis) [reply .github/workflows/push.yml]: I reset this workflow mostly back to the way that it was.

**Comment by stdavis** [comment]
@steveoh This is ready for another look.

**Review by steveoh — APPROVED** [review]
this looks like it might work. I'd like to see it try

**Pull request merged**